src/main/java/core/basesyntax/service/ReportCreatorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    private static final String REPORT_TITLE = "fruit,quantity";
+
+    @Override
+    public List<String> createReport(Map<String, Integer> fruits) {
```

Comment:
```suggestion
    public String createReport(Map<String, Integer> fruits) {
```
better make a ready report to just write it to a file